src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/AllowReversePInvokeCallsAttributeTests.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Reflection;
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public class AllowReversePInvokeCallsAttributeTests
+    {
+        [AllowReversePInvokeCalls]
+        private int Func(int a, int b)
+        {
+            return a + b;
+        }
+
+        [Fact]
+        public void Exists()
+        {
+            var type = typeof(AllowReversePInvokeCallsAttributeTests);
+            var method = type.GetTypeInfo().DeclaredMethods.Single(m => m.Name == "Func");
+            var attr = method.GetCustomAttributes(typeof(AllowReversePInvokeCallsAttribute), false).Single(a => a is AllowReversePInvokeCallsAttribute);
+            Assert.NotNull(attr);
```

Comment:
Here and in other tests where you follow this pattern, NotNull will never fail... if the attribute couldn't be found, Single would throw, so the only way you'll get to the NotNull call is if attr is not null.